api/src/pcapi/core/external/sendinblue.py

```diff
@@ -247,6 +255,36 @@ def format_user_attributes(attributes: attributes_models.UserAttributes | attrib
     }
 
 
+def format_pro_attributes(attributes: attributes_models.UserAttributes | attributes_models.ProAttributes) -> dict:
```

Comment:
Ces attributs sont remplis au-dessus à partir de `ProAttributes` dans ce qu'on envoie actuellement, mais ne sont pas repris ici :
- `DEPARTMENT_CODE`
- `MARKETING_EMAIL_SUBSCRIPTION`
- `POSTAL_CODE`
- `IS_EAC_MEG`

Oubliés ou y a-t-il une raison particulière ?

Reply:
C'est pas la première différence que je trouve (cf has_individual_offers); je vais confirmer ça avec Nine

Reply:
Autre remarque, la signature de la fonction ne devrait attendre que des `ProAttributes`, non ?
```suggestion
def format_pro_attributes(attributes_models.ProAttributes) -> dict:
```